src/components/clusters/ClusterPage.tsx

```diff
@@ -196,19 +156,13 @@ const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
   if (cluster) {
     return (
       <AlertsContextProvider>
-        <ClusterDefaultConfigurationProvider loadingUI={loadingUI} errorUI={errorUI}>
-          {getContent(cluster)}
-          <CancelInstallationModal
-            isOpen={cancelInstallationModalOpen}
-            onClose={() => setCancelInstallationModalOpen(false)}
-            clusterId={cluster.id}
-          />
-          <ResetClusterModal
-            isOpen={resetClusterModalOpen}
-            onClose={() => setResetClusterModalOpen(false)}
-            cluster={cluster}
-          />
-        </ClusterDefaultConfigurationProvider>
+        <HostDialogsContextProvider>
```

Comment:
This is very confusing because of the `HostDialogsContextProvider` name which now includes cluster specific dialogs. I also suspect that nesting these provides may cause more problems?

Reply:
Right, what about renaming it? I don't see a point in having it specific, the IDs can be unique. Will double-check nesting, would be unintended.

Reply:
ok renaming should be fine I think.

Reply:
Since we'll have `DialogsContextProvider` which we use to track all dialogs state, we can render it just at `ClusterPage`, `AI Detail card` (+ other OCM related components if we have such) and `Clusters`.

We can remove it from HostsTable

Reply:
done